src/vtcomposite.cpp

```diff
@@ -677,29 +705,22 @@ struct InternationalizeWorker : Napi::AsyncWorker
                 vtzero::layer_builder lbuilder{tbuilder, layer.name(), layer.version(), layer.extent()};
                 while (auto feature = layer.next_feature())
                 {
-                    std::vector<std::string> worldviews;  // ren mbx_worldviews
+                    std::vector<std::string> worldviews_to_create;
                     bool has_mbx_worldview = false;
-
                     bool name_was_set = false;
                     vtzero::property_value name_value;
 
                     // accumulate final properties (except _mbx_worldview translation to worldview) here
-                    property_list properties(feature.num_properties);
+                    std::vector<std::pair<std::string, vtzero::property_value>> properties;
```

Comment:
Earlier, I _meant_ to reserve space, given we know pretty closely how many properties will be added. What do you think? `properties.reserve(feature.num_properties)`

Reply:
Oops my bad - I didn't mean to remove this long-term. Originally I encountered an error as-is because `num_properties` is a function not a static property. But now if I update to `properties(feature.num_properties())` we get some funky errors in tests:

```
    expected: [ 'name_en', 'name_fr', 'name_local', 'name', 'population' ]
    actual:   [ '', 'name_en', 'name_fr', 'name_local', 'name', 'population' ]

    expected: |-
      { name: 'Germany', name_local: 'Germany', name_en: 'Germany', name_fr: 'Allemagne' }
    actual: |-
      { : null, name: 'Germany', name_en: 'Germany', name_fr: 'Allemagne', name_local: 'Germany' }
```

I tried updating to `properties(feature.num_properties() - 1)` in case it was just an off-by-one issue, but that doesn't appear to resolve it.